ipn/ipnstate/ipnstate.go

```diff
@@ -33,10 +33,6 @@ type Status struct {
 	//  "Starting", "Running".
 	BackendState string
 
-	// TailnetName is the name of the network that's currently in
```

Comment:
This field was never released in a public build, so it seems ideal to move it before folks rely on it.